tests/float/Makefile

```diff
@@ -9,4 +7,5 @@ endif
 
 TEST_ON_CI_WHITELIST += all
 
+include ../Makefile.tests_common
```

Comment:
Here also we use `BOARD`, a conditional definition of `CFLAGS` could be done, or just changing the C file would even be better…